From 6e3928154538f8935d9d6379e3db40daeb1aedcf Mon Sep 17 00:00:00 2001
From: James Feist <james.feist@linux.intel.com>
Date: Thu, 19 Sep 2019 10:15:19 -0700
Subject: [PATCH] Re-Enable KCS

Phosphor-isolation isolates too much and seems to
make KCS not work. This removes the lines from that
patch that seem to be of question.

Tested: AC/Cycled, ipmitool raw 6 1 works

Signed-off-by: James Feist <james.feist@linux.intel.com>

---
 board/aspeed/ast-g5/ast-g5.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/board/aspeed/ast-g5/ast-g5.c b/board/aspeed/ast-g5/ast-g5.c
index ead2e1bb63..00bd92ae5f 100644
--- a/board/aspeed/ast-g5/ast-g5.c
+++ b/board/aspeed/ast-g5/ast-g5.c
@@ -41,11 +41,6 @@ int board_init(void)
 	bool sdmc_unlocked;
 	u32 val;
 
-	/* iLPC2AHB */
-	val = readl(AST_SCU_BASE + AST_SCU_HW_STRAP1);
-	val |= SCU_HW_STRAP_LPC_DEC_SUPER_IO;
-	writel(val, AST_SCU_BASE + AST_SCU_HW_STRAP1);
-
 	val = readl(AST_LPC_BASE + AST_LPC_HICRB);
 	val |= LPC_HICRB_ILPC2AHB;
 	writel(val, AST_LPC_BASE + AST_LPC_HICRB);
